tests/integrations/test_cases/execute_at_address.py

```diff
@@ -29,24 +41,12 @@
-            "calldata": "18160ddd",
+            "calldata": keccak_256("totalSupply()".encode()).hexdigest()[:8],
             "stack": "",
             "memory": "",
             "return_value": "0000000000000000000000000000000000000000000000000000000000000000",
         },
         "id": "solmate_erc20_totalSupply",
         "marks": [pytest.mark.SolmateERC20],
     },
-    {
```

Comment:
Why did you delete this?

Reply:
furthermore, in the context of `execute_at_address`, it should revert as the calldata is not long enough (2 bytes instead of 4)

Reply:
right!